Jenkinsfile

```diff
@@ -1,5 +1,12 @@
 #!/usr/bin/env groovy
-common {
-  slackChannel = '#clients-eng'
-  upstreamProjects = 'confluentinc/rest-utils'
+
+docker_oraclejdk8 {
+  dockerPullDeps = ['confluentinc/cp-base']
+  dockerPush = true
+  dockerRegistry = '368821881613.dkr.ecr.us-west-2.amazonaws.com/'
```

Comment:
can we get some of these (especially things like the docker registry) as default values inherited from `jenkins-common` so we don't duplicate the same info a bunch of times?

Reply:
Good idea, let's stop this from spreading.